src/test/java/com/clouway/cuse/SearchEngineContractTest.java

```diff
@@ -979,6 +979,66 @@ public void searchEmptyString() throws Exception {
     assertThat(result.size(), is(3));
   }
 
+  @Test
+  public void registerDocumentsInBulkChunksWithEmployees() throws Exception {
+    List employeeList = generateGiantListOfEmployeesNamedFooBar(402);
+    storeAllEmployees(employeeList);
+    store(new User(2L));
+    List result = searchEngine.search(Employee.class).returnAll().now();
+    assertThat(employeeList.size(), is(result.size()));
+  }
+
+  @Test
+  public void registeringDocumentsInBulkChunksWithUsers() throws Exception {
+    List<User> userList = new LinkedList<User>() {{
```

Comment:
Instead of `new LinkedList` you can use:
```java
List<User> userList = Lists.newArrayList(new User(1L), new User(2L), new User(3L));
```
same way with all the usages below.